### PR TITLE
Create eip155-107.json

### PR DESCRIPTION
**Pull request by nova-network-inc**
Add Nebula Testnet

### CODE REVIEW

File: _data/chains/eip155-107.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Nebula Testnet",
+  "chain": "NBX",
+  "icon": ""nebulatestnet",
```
Comment (ligi) [inline _data/chains/eip155-107.json:4]:
```suggestion
  "icon": "nebulatestnet",
```

File: _data/chains/eip155-107.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "symbol": "NBX",
+    "decimals": 18
+  },
+  "infoURL": "https://novanetwork.io/",
+  "shortName": "nbx",
+  "chainId": 107,
+  "networkId": 107,
+  "explorers": [{
+    "name": "nebulatestnet",
+    "url": "http://explorer.novanetwork.io/?network=Nebula%20Testnet",
```
Comment (ligi) [inline _data/chains/eip155-107.json:21]:
```suggestion
    "url": "https://explorer.novanetwork.io/?network=Nebula%20Testnet",
```

**Comment by ligi** [comment]
The explorer seems to not work?

**Comment by nova-network-inc** [comment]
> The explorer seems to not work?
It only works with http at the moment. We have just deployed the SSL certificates. Just drop the 's' and you should be able to access it, as it's working fine.

**Comment by nova-network-inc** [comment]
Have just deployed the nodes, and geth isn't directly compatible with the https protocol. We have already set up the reverse DNS for the RPCs and have just requested the SSL certificates to be signed by DigiCert through Cloudflare. Just waiting for that to kick in, and it should be accessible via HTTPS.

**Comment by nova-network-inc** [comment]
And just in addition to that, the explorer has been forked from Expedition https://github.com/xops/expedition and our version of the code is also open source and accessible here https://github.com/nova-network-inc/nova-explorer

**Comment by nova-network-inc** [comment]
If it's a problem we can drop the explorer info on both chains 87 and 107, and commit again when the HTTPS is working. Just let us know.

**Comment by ligi** [comment]
yea - let's do that

**Comment by nova-network-inc** [comment]
Can you remove it, or do you want me to recommit?

File: _data/chains/eip155-107.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "infoURL": "https://novanetwork.io/",
+  "shortName": "nbx",
+  "chainId": 107,
+  "networkId": 107,
+  "explorers": [{
+    "name": "nebulatestnet",
+    "url": "https://explorer.novanetwork.io/?network=Nebula%20Testnet",
+    "standard": "EIP3091"
+  }]
```
Comment (ligi) [inline _data/chains/eip155-107.json:23]:
```suggestion
  "networkId": 107
```

**Comment by nova-network-inc** [comment]
Hi, I saw it has failed on the build/push, but I couldn't find anything wrong with it. Sorry, but could you perhaps shout some light on it for me? Thanks.

**Review by Gunawanwahyudi21 — COMMENTED** [review]
Good project and nice project, go to support

**Review by Gunawanwahyudi21 — COMMENTED** [review]
Good project and nice project, go to support

**Review by ligi — CHANGES_REQUESTED** [review]
The icon nebulatestnet does not exist

**Comment by nova-network-inc** [comment]
> The icon nebulatestnet does not exist

Nope, it's all right... Perhaps you just overlooked it, or if there's something wrong I'm not seeing, please let me know. Cheers.

**Pull request merged**